rsconnect/bundle.py

```diff
@@ -366,10 +366,10 @@ def make_html_manifest(
 def make_notebook_html_bundle(
     filename: str,
     python: str,
-    image: str,
     hide_all_input: bool,
     hide_tagged_input: bool,
-    check_output: typing.Callable,  # used to default to subprocess.check_output
+    image: str = None,
+    check_output: typing.Callable = subprocess.check_output,  # used to default to subprocess.check_output
 ) -> typing.IO[bytes]:
     # noinspection SpellCheckingInspection
     if check_output is None:
```

Comment:
This check is probably not needed, since the production code uses defaults and test code will inject a value.